.github/workflows/node-ci.yml

```diff
@@ -30,6 +30,27 @@ jobs:
       with:
         format: json
 
+    - name: Check for Changeset
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const inputFiles = JSON.parse(`${{ steps.changed.outputs.added_modified }}`);
+          core.debug("Changed files:", inputFiles);
+
+          const hasChangeset = inputFiles.some(name => {
+            return /^\.changeset\/\w+-\w+-\w+\.md/.test(name);
+          });
```

Comment:
I would make this regex more general since a person could manually add a .md file to the .changeset directory (or they could rename it if it was automatically generated to have a more descriptive name).  Maybe something like `/^\.changeset\/.*\.md$/`.